frontend/ReleaseDashboard/TestDashboard.svelte

```diff
@@ -221,6 +223,24 @@
         }
     };
 
+    const shouldFilterByUser = function (user, test, group = undefined, tests = undefined, type = "test") {
+        if (!user) return false;
+        console.log(test);
+        switch (type) {
+        case "test":
+            return getAssigneesForTest(test.test.id, test.test.group_id, test.last_runs) != user.id;
+        case "group":
+            return (assigneeList.groups?.[group.id]?.[0] ?? "") != user.id && tests.filter(v => !shouldFilterByUser(user, v)).length == 0;
```

Comment:
do we need `assigneeList.groups?.[group.id]?.[0] ?? "") != user.id` part?

Reply:
I think there's a bug related to group assignees and TestDashboard currently (which I may or may have not yet fixed), but yes, the group itself can have an assignee.